tests/trainer/optimization/test_multiple_optimizers.py

```diff
@@ -61,3 +63,79 @@ def configure_optimizers(self):
     # test loss are properly reduced
     assert torch.abs(trainer.callback_metrics["loss_2_epoch"] - torch.FloatTensor(model.loss_2).mean()) < 1e-6
     assert torch.abs(trainer.callback_metrics["loss_1_epoch"] - torch.FloatTensor(model.loss_1).mean()) < 1e-6
+
+
+def test_multiple_optimizers(tmpdir):
```

Comment:
Not sure to understand those tests. Would be better to log something and make sure the EpochResultStore is properly populated.

Reply:
These are from another file also for multioptimizer tests. I merged the files into one.

Reply:
Ok !